components/Challenge.yml

```diff
@@ -0,0 +1,65 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    CreateChallengeResult:
+      type: object
+      additionalProperties: false
+      description: Object containg a challenge
+      properties:
+        challenge:
+          type: string
+          description: The challenge value
+    ChallengeVerificationMetadata:
+      type: object
+      additionalProperties: false
+      description: Metadata about the verification of options.challenge.
+      properties:
+        verified:
+          type: string
+          description: Whether verification of the challenge was successful
+        uses:
```

Comment:
```suggestion
        firstVerified:
          type: string
          description: dateTimeStamp when the challenge was first verified.
        uses:
```

Reply:
We need to specify that expiry information needs to be specified somewhere in the specification, such as in the instance configuration (or some other machine-readable location?). We should discuss if/where that information should be and the use case for using the `expiry`, and `firstVerified` -- perhaps raise an issue to work through this (should exchanges expire? should challenges expire?).

Reply:
#374 addresses this.